packages/components/src/components/Note/Note.tsx

```diff
@@ -5,26 +5,29 @@ import { Icon } from '../Icon';
 import { FONT_SIZE, FONT_WEIGHT } from '../../config/variables';
 
 const Row = styled.div`
-    align-items: center;
     display: flex;
     gap: 6px;
 `;
 
-const P = styled.p`
-    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+const InfoIcon = styled(Icon)`
+    margin-top: 1px;
+`;
+
+const P = styled.p<{ $color?: string }>`
+    color: ${({ $color, theme }) => $color || theme.TYPE_LIGHT_GREY};
```

Comment:
perhaps it's clearer to define the default color in component props?

Reply:
I moved the color definition to the component so that it is shared between the `Icon` and `P`.